CHANGELOG.md

```diff
@@ -91,6 +91,7 @@
 
 ### Mixin
 
+* [ENHANCEMENT] Alerts: Add MimirRingMembersMismatch firing when a component does not have the expected number of running jobs. #2404
```

Comment:
We're used to group CHANGELOG entries by type (see https://github.com/grafana/mimir/tree/main/docs/internal/contributing#changelog).

Reply:
Ack, thanks